### PR TITLE
Fixing nxos_vrrp status bug

### PR DESCRIPTION
**Pull request by GGabriele**
##### ISSUE TYPE
- Bugfix Pull Request
##### COMPONENT NAME

nxos_vrrp
##### ANSIBLE VERSION

```
ansible 2.1.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = xxx
```
##### SUMMARY

Added a param to specify the VRRP process status, which would always be 'shutdown' by default.

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks again to @GGabriele. This module is going into community review.

We notice that you are one of the maintainers (@privateip, @GGabriele) of this module, so when you think it's ready, please comment "shipit" and we will consider it for merging.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by GGabriele** [comment]
shipit

**Comment by gregdek** [comment]
Thanks @GGabriele. Since you are one of the maintainers (@privateip, @GGabriele) of this module and you commented with "shipit", we are marking this PR for inclusion.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
Thanks @GGabriele for this PR. This PR requires revisions, either because it fails to build or by reviewer request. Please make the suggested revisions. When you are done, please comment with text 'ready_for_review' and we will put this PR back into review.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@GGabriele A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes. When you do address the issues, please respond with ready_for_review in your comment, so that we can notify the maintainer.

[This message brought to you by your friendly Ansibull-bot.]

File: network/nxos/nxos_vrrp.py [hunk 1]
```diff
@@ -57,6 +57,12 @@
             - clear text authentication string
         required: false
         default: null
+    admin_state:
+        description:
+            - Used to enable or disable the VRRP process
+        required: false
```
Comment (gundalow) [inline network/nxos/nxos_vrrp.py:63]: Can you please add
`version_added: "2.2"`
at the same level as `description` and `required`

File: network/nxos/nxos_vrrp.py [hunk 2]
```diff
@@ -21,7 +21,7 @@
 ---
 
 module: nxos_vrrp
-version_added: "2.1"
```
Comment (gundalow) [inline network/nxos/nxos_vrrp.py:24]: Please revert this change, the `version_added` field should go under `admin_state`

**Comment by gundalow** [comment]
needs_revision
